### PR TITLE
feature/APSPlanner rename to ApsPlanner

### PR DESCRIPTION
**Pull request by Voyz**
- renamed APSPlanner to ApsPlanner
- added APSPlanner wrapper class to warn of the deprecation

### CODE REVIEW

File: databay/planners/aps_planner.py [hunk 1]
```diff
@@ -194,4 +198,10 @@ def running(self):
         return self._scheduler.state == STATE_RUNNING
 
     def __repr__(self):
-        return 'APSPlanner(threads:%s)' % (self._threads)
+        return 'ApsPlanner(threads:%s)' % (self._threads)
+
+
+class APSPlanner(ApsPlanner): # pragma: no cover
+    def __init__(self, *args, **kwargs):
+        warnings.warn('APSPlanner was renamed to ApsPlanner and will be permanently changed in version 1.0', DeprecationWarning)
```
Comment (cdpierse) [inline databay/planners/aps_planner.py:206]: Might be worth also mentioning what version this change was brought in on in the warning. 
i.e.
`warnings.warn('APSPlanner was renamed to ApsPlanner in version 0.1.x and will be permanently changed in version 1.0', DeprecationWarning)`

**Review by cdpierse — CHANGES_REQUESTED** [review]
Everything looks really good besides possibly adding some extra detail about the version the change was implemented in the depreciation warning.

**Comment by Voyz** [comment]
Good catch! Done 👍

**Comment by Voyz** [comment]
Hold on, it just occurred to me: did you mean to include the actual version number, `0.1.7` instead of `0.1.x`? It would make more sense I guess, although may be tricky to nail that number down accurately before a release if releases get bumped or features get bumped. What do you recon?

**Comment by cdpierse** [comment]
Yes I meant the actual version like `0.1.7` if we are in a situation where it's really clear to mark where the change occurred. Otherwise we might be better off sticking `0.1.7`.

**Comment by Voyz** [comment]
Thanks for the review 👍

**Pull request merged**